assets/js/blocks/price-filter/block.json

```diff
@@ -0,0 +1,46 @@
+{
+	"name": "woocommerce/price-filter",
+	"version": "1.0.0",
+	"title": "Filter Products by Price",
+	"description": "Allow customers to filter the products by choosing a lower or upper price limit. Works in combination with the All Products block.",
+	"category": "woocommerce",
+	"keywords": [ "WooCommerce" ],
+	"supports": {
+		"html": false,
+		"multiple": false,
+		"color": {
+			"text": true,
+			"background": false
+		}
+	},
+	"example": {
+		"attributes": {
+			"isPreview": true
+		}
+	},
+	"attributes": {
+		"showInputFields": {
+			"type": "boolean",
+			"default": true
+		},
+		"showFilterButton": {
+			"type": "boolean",
+			"default": false
+		},
+		"heading": {
+			"type": "string",
+			"default": "Filter by price"
+		},
+		"headingLevel": {
+			"type": "number",
+			"default": 3
+		},
+		"className": {
+			"type": "string",
+			"default": ""
+		}
+	},
+	"textdomain": "woo-gutenberg-products-block",
+	"apiVersion": 2,
+    "$schema": "https://schemas.wp.org/trunk/block.json"
```

Comment:
Could we also add this `$schema` property to the other `block.json` files in the PR?
Also, super picky, but I see a different indentation at the beginning of this line 🤔

Reply:
> Also, super picky, but I see a different indentation at the beginning of this line

Total not! Thanks for pointing this out.